Tests/Tests/RegistrationTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+
+namespace Tests.Tests
+{
+    public class RegistrationTest : BasicTest
+    {
+        [Test]
+        public void NewAccountCreationTest()
+        {
+            var page = mainPage
+                .GoToLoginPage()
+                .GoToRegistrationPage();
+
+            Assert.AreEqual(page.PageTitle.Text, "Регистрация");
+            
+                page.SignUp();
+
+            var accountPage = header.GoToAccountPage();
+
+            Assert.IsTrue(accountPage.LogoutBtn.Displayed);
+            Assert.AreEqual(accountPage.PageTitle.Text, "Личный кабинет");
```

Comment:
It would be great if your test checked the existence of a newly created account: e.g.: you can log in and will see your private cabinet.

Reply:
Test was edited.